{{cookiecutter.project_slug}}/.watchmanconfig

```diff
@@ -0,0 +1,13 @@
+{
+  "ignore_dirs": [
+    "node_modules",
+    ".envs",
+    "bin",
+    "lib",
+    "include",
+    ".git",
+    ".idea",
+    "templates",
+    "static_assets"
+  ]
```

Comment:
Check if using Docker. If the dev is, you should add "compose".

Reply:
Done